app/routes/index.js

```diff
@@ -5,12 +5,15 @@ var router = express.Router();
 var campaignController = require("../middleware/campaign");
 var influencerController = require("../middleware/influencer");
 var apiRoutes = require('./api')
+var secrets = require('../../secrets');
+var squareProxy = require('express-http-proxy');
 
-router.get('/', function(request, response) {
+router.get('/', function(request, response, next) {
     if(request.user){
         response.redirect('/home');
     } else {
-        response.redirect('/login');
+        proxyResponse = squareProxy(secrets.marketing_site_url);
```

Comment:
I kept the hostname of the marketing website (pointed to by the reverse proxy) in the secrets file, and since I forgot to update it, the reverse proxy wasnt working in staging. Since its ok to put values like the marketing website hostname in source control, and thus in a separate, source-controlled config file, I think its a good idea to separate some non-sensitive stuff in a config file.